zerver/openapi/zulip.yaml

```diff
@@ -12125,6 +12125,48 @@ paths:
                         description: |
                           An example JSON error response for when the user doesn't have permission
                           to subscribe other users to streams and `stream_ids` is not empty.
+  /invites/{prereg_id}:
+    delete:
+      operationId: revoke-email-invitation
+      summary: Revoke email invitation
+      tags: ["invites"]
+      description: |
+        Revoke an [invitation](/help/invite-new-users#send-email-invitations)
+        that has been sent to the email address of the invitee.
+
+        The non-admin users can only revoke the invitation that they have sent. The
+        administrators can revoke any invitation except the ones that have
+        `invited_as` parameter as `100` (i.e. user created after the acceptance of
+        the invitation will be assigned the [organization-level role](/help/roles-and-permissions)
+        as owner). Only the owners in the organization can revoke such invitations.
```

Comment:
So, when describing a group of people in general, one doesn't need to use the definite article ("the").

I wonder if this can be simplified to just state that a user can only revoke invitations that they can manage, and link to the section in the help documentation about managing invitations: https://zulip.com/help/invite-new-users#manage-pending-invitations